roles/nfs_volumes/library/partitionpool.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+"""
+Ansible module for partitioning.
```

Comment:
@detiber Provisioning a cluster is a manual task at present.  The goal behind this module was to either automate a tedious task for an admin (provision an OpenShift cluster with NFS storage) or provide some inspiration/insight into how the admin can do it themselves.